Manager/src/test/java/io/raspberrywallet/manager/ManagerTest.java

```diff
@@ -0,0 +1,81 @@
+package io.raspberrywallet.manager;
+
+import io.raspberrywallet.manager.bitcoin.Bitcoin;
+import org.bitcoinj.crypto.MnemonicException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+import static io.raspberrywallet.manager.Utils.println;
+
+class ManagerTest {
+    private static Manager manager;
+    static Bitcoin bitcoin;
+
+    @BeforeAll
+    static void setup() {
+        bitcoin = Mockito.mock(Bitcoin.class);
+        manager = new Manager(bitcoin);
+    }
+
+    @Test
+    void ping() {
+        assert manager.ping().equals("pong");
```

Comment:
It would be better to use assertions from JUnit for consistency:
https://stackoverflow.com/questions/2966347/assert-vs-junit-assertions

Reply:
Right